src/frontend/dsl/lb.rs

```diff
@@ -201,3 +201,187 @@ impl<F: Clone + Debug> LookupBuilder<F> for LookupTableBuilder<F> {
         table.build(self.src, self.enable)
     }
 }
+
+#[cfg(test)]
+mod test {
+
+    use halo2_proofs::halo2curves::bn256::Fr;
+    use halo2curves::ff::Field;
+
+    use crate::ast::StepType;
+    use crate::ast::Expr;
+
+    use super::*;
+
+    #[test]
+    fn test_in_place_lookup_builder_build() {
+        let in_place_lookup_builder = InPlaceLookupBuilder::<Fr>::default();
+        let annotation = in_place_lookup_builder.lookup.annotation;
+        let exprs = in_place_lookup_builder.lookup.exprs;
+        let enable = in_place_lookup_builder.lookup.enable;
+        assert_eq!(annotation, "");
+        assert_eq!(format!("{:?}",exprs), "[]");
```

Comment:
@motemotech I think you can check exprs.len() eq 0, without converting to string